ui/component/common/markdown-preview.jsx

```diff
@@ -70,6 +78,25 @@ const SimpleLink = (props: SimpleLinkProps) => {
   return <a title={title}>{children}</a>;
 };
 
+// ****************************************************************************
+// ****************************************************************************
+
+function ZoomableImage({ ...imgProps }) {
+  const onClick = () => {
+    window.app.store.dispatch(
```

Comment:
Could this just be turned into a regular component that connects to the store?

I would like to avoid using `window.app.store.dispatch`